sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -481,19 +481,26 @@ class PlanResolutionSuite extends AnalysisTest {
       "other" -> "20")
 
     parseAndResolve(sql) match {
-      case create: CreateV2Table =>
-        assert(create.catalog.name == "testcat")
-        assert(create.tableName == Identifier.of(Array("mydb"), "table_name"))
+      case create: CatalystCreateTable =>
+        assert(create.name.asInstanceOf[ResolvedDBObjectName].catalog.name == "testcat")
+        assert(create.name.asInstanceOf[ResolvedDBObjectName].nameParts.mkString(".") ==
+          "mydb.table_name")
         assert(create.tableSchema == new StructType()
             .add("id", LongType)
             .add("description", StringType)
             .add("point", new StructType().add("x", DoubleType).add("y", DoubleType)))
         assert(create.partitioning.isEmpty)
-        assert(create.properties == expectedProperties)
```

Comment:
I think we don't need to test it now as it's not part of the resolution anymore.

Reply:
do we have end-to-end test for CREATE TABLE? we can test it there.

Reply:
We have an end to end test here for table properties https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala#L123